frontend/src/Onboarding/RedirectScreen.tsx

```diff
@@ -90,17 +80,7 @@ export const RedirectScreen: React.FC<Props> = ({ redirectUrl }) => {
         }); // set persisting cookie for all paths
 
         fetchUser(cookie).then(response => {
-          dispatch(setFullNameAction(response.user.username));
-          dispatch(setGraduationYearAction(response.user.graduationYear));
-          dispatch(setAcademicYearAction(response.user.academicYear));
-          const maj = findMajorFromName(response.user.major, majors);
-          if (maj) {
-            dispatch(setDeclaredMajorAction(maj));
-          }
-          dispatch(setUserIdAction(response.user.id));
-          dispatch(setEmailAction(response.user.email));
-          dispatch(setUserCoopCycleAction(response.user.coopCycle));
-          dispatch(setIsAdvisorAction(response.user.isAdvisor));
+          dispatch(setUserAction(response.user));
```

Comment:
We can just set the whole user object instead of setting each field individually

Reply:
Keep in mind this requires that the fields in the user object in redux must match the user object returned from the backend

Reply:
don't think this will work for transfer/completed courses. What happens if we pass the response and it doesn't match, will it just not set those fields or will it error?